src/i18n/locales/en.json

```diff
@@ -92,5 +92,7 @@
   "blog.categories.vulnerability": "Vulnerabilities",
   "blog.categories.announcements.description": "Blog category for Node.js announcements",
   "blog.categories.release.description": "Blog category for Node.js Release announcements",
-  "blog.categories.vulnerability.description": "Blog category regarding Security Vulnerabilities"
+  "blog.categories.vulnerability.description": "Blog category regarding Security Vulnerabilities",
+  "components.downloadToggle.ltsVersions": "LTS Versions",
```

Comment:
Doesn't look like this file stayed sorted, but might be good to keep these up with the other download strings around `components.download*`

Reply:
I was told in a previous PR to add my translations to the bottom instead of somewhere in the middle even if there was already a section as it reduces merge conflicts but I’m happy to move them to the correct section if you’d prefer

Reply:
Sorry, I wasn't sure if there was any convention for those yet. Since someone said the other way around before, it's definitely something better suited for a separate issue

Reply:
> Sorry, I wasn't sure if there was any convention for those yet. Since someone said the other way around before, it's definitely something better suited for a separate issue

I think it is written within `TRANSLATING.md`? But if not let me update that!